src/app/search/search.component.scss

```diff
@@ -196,41 +96,9 @@ mat-expansion-panel mat-expansion-panel-header[aria-expanded='false'] {
   text-decoration: underline;
 }
 
-// Form field box
-:host .mat-expansion-panel ::ng-deep .mat-form-field-wrapper {
-  margin: 0 0.75rem;
-  padding-bottom: 0.8rem;
-  line-height: 3.6rem;
-}
-
-// Form field label
-:host
-  .mat-expansion-panel
-  ::ng-deep
-  .mat-form-field-wrapper
-  .mat-form-field-flex
-  .mat-form-field-infix
-  .mat-form-field-label-wrapper
-  .mat-form-field-label {
-  top: 1.8rem;
-}
-
-// Form field inner
-:host .mat-expansion-panel ::ng-deep .mat-form-field-wrapper .mat-form-field-infix {
-  border: 0rem;
-  padding: 0rem;
-}
-
-// Form field text animation re-implementation
-:host ::ng-deep .mat-form-field-appearance-outline.mat-form-field-can-float.mat-form-field-should-float .mat-form-field-label {
-  transform: translateY(-1.1em) scale(0.75);
-  width: 133.33333%;
-  top: 1.84375em;
-}
-
```

Comment:
These styles made the outline form field smaller than the default appearance. In this PR, we achieve the same effect by setting the form-field density to -4 in `_material-overrides.scss`. I would recommend not targeting internal Material CSS classes that are deeply nested because it's hard to maintain and it may misalign things, causing a domino effect of adding more internal Material styles to fix the misalignment.

Before: 
![image](https://github.com/user-attachments/assets/bdd9cbfb-a08e-4634-9089-d99fc3e735e2)

After:
![image](https://github.com/user-attachments/assets/6d1ad19e-5168-4782-a0a9-e48d9c6eb71e)